var.tfvars

```diff
@@ -1,86 +1,80 @@
-### Configure the OpenStack Provider
-auth_url                    = "https://<HOSTNAME>:5000/v3/"
-user_name                   = ""
-password                    = ""
-tenant_name                 = "tenant_name"
+### PowerVC Details
+auth_url                    = "<https://<HOSTNAME>:5000/v3/>"
+user_name                   = "<powervc-login-user-name>"
+password                    = "<powervc-login-user-password>"
+tenant_name                 = "<tenant_name>"
 domain_name                 = "Default"
 openstack_availability_zone = ""
 
-### Configure the Instance details
-network_name                = "network_name"
-#network_type               = "SRIOV"
-#scg_id                      = "df21cec9-c244-4d3d-b927-df1518672e87"
+network_name                = "<network_name>"
+
+### OpenShift Cluster Details
+
+bastion                     = {instance_type    = "medium", image_id    = "<image-uuid-rhel>"}
+bootstrap                   = {instance_type    = "medium", image_id    = "<image-uuid-rhcos>",  "count"   = 1}
+master                      = {instance_type    = "medium", image_id    = "<image-uuid-rhcos>",  "count"   = 3}
+worker                      = {instance_type    = "large",  image_id    = "<image-uuid-rhcos>",  "count"   = 2}
+
+
 rhel_username               = "root"
-#keypair_name                = "mykeypair"
 public_key_file             = "data/id_rsa.pub"
 private_key_file            = "data/id_rsa"
-private_key                 = ""
-public_key                  = ""
-rhel_subscription_username  = ""
-rhel_subscription_password  = ""
+rhel_subscription_username  = "<subscription-id>"          #Leave this as-is if using CentOS as bastion image
```

Comment:
Value "<subscription-id>" wont work for centos.. should be "" or code needs to be updated.

Reply:
Ideally the code should also handle empty string. However currently this should work since it's not empty string. Do you see any issue with this ?

Reply:
My concern is if user is using CentOS  and leave `rhel_subscription_username` as is then  https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/modules/1_bastion/bastion.tf#L162 will go ahead and register the centos machine which we don't want.